src/composables/useLeaflet.ts

```diff
@@ -134,8 +135,14 @@ export const useLeaflet = (
 
         resetMaxBoundsAndZoom();
 
+        // please keep addition of empty layer first
+        // more info: https://github.com/mrc-ide/arbomap/pull/31#issuecomment-2110480157
```

Comment:
in summary:
* if we try to add a polyline after a geojson to a map with no layers it errors
* if we add a geojson after a polyline to a map it is fine but that means the country outline would be underneath the geojson and it looks ugly
* if we add a polyline to the map first and then add any layer afterwards it is also fine

I think it must be some sort of weird bug with the implementation of polyline so i have chosen a polyline for the empty layer and this layer will never be removed from the map and will only be added once

Reply:
Thanks! Can you link to here instead of to the longer comment? Which, for posterity, was https://github.com/mrc-ide/arbomap/pull/31#issuecomment-2110480157

Reply:
done